### ISSUE TITLE
Accessibility issue - 1.3.1 Info and Relationships

### ISSUE THREAD

**Issue by phucduong86** [shiffman/shiffman.net#80]
Results for: (https://shiffman.net)
Findings: Recommend using CSS to achieve text elements styling instead of heading elements (right sidebar).

Results for: (https://shiffman.net/books/)
Findings: Headings on the right panel are not level one heading they should be h2 or h3 and achieve styling via CSS.
"Books" should be level 1 heading.

Results for: (https://shiffman.net/videos/)
Findings: Heading markup for elements that are not headings (right quick links). Recommend achieving styling via CSS.

Results for: (https://shiffman.net/learning/)
Findings: Structurally "Learning" heading should be level 1.

Results for: (https://shiffman.net/a2z/)
Findings: The links under "NOTES" are part of a list and should be programmed as such.

Results for: (https://shiffman.net/a2z/closures/)
Findings: Structurally "Closure" heading is one level below "Programming from A to Z" heading so it should be h2. Means all the other headings under "Closure" should be a level lower.

Results for: (https://shiffman.net/a2z/cfg/)
Findings: Same issue as https://shiffman.net/a2z/closures
